src/pages-and-resources/live/Settings.jsx

```diff
@@ -70,9 +76,18 @@ const LiveSettings = ({
     await dispatch(saveLiveConfiguration(courseId, values));
   };
 
+  const configureZoomGlobalSettingsIfExists = async (id) => {
+    await dispatch(configureZoomGlobalSettings(id));
+  };
+
   useEffect(() => {
+    if (!isZoomGlobalCredSet && isZoomBtnClicked) {
+      configureZoomGlobalSettingsIfExists(courseId);
+      setIsZoomBtnClicked(false);
+      dispatch(updateIsZoomGlobalCredSet({ isZoomGlobalCredSet: true }));
+    }
     dispatch(fetchLiveData(courseId));
-  }, [courseId]);
+  }, [courseId, isZoomBtnClicked]);
```

Comment:
This action does not require useEffect. Set `updateIsZoomGlobalCredSet`  to true when a user clicks on `global config button`. After that send an API call. This way the user will not face any laggy UI. If API sends an error response then reset the button value.